metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery.py

```diff
@@ -245,7 +246,8 @@ def __init__(self, ctx: PipelineContext, config: BigQueryV2Config):
                 self.config.get_policy_tag_manager_client()
             )
 
-        self.sql_parser_schema_resolver = self._init_schema_resolver()
+        self.sql_parser_schema_resolver = self._init_schema_resolver(self.platform)
+        self.gcs_parser_schema_resolver = self._init_schema_resolver("gcs")
```

Comment:
why do we have gcs schema resolver? this doesn't really make sense

Reply:
hi @hsheth2 It helps us gather all GCS URNs and identify source_urn for lineage extraction.

Reply:
do we have schemas for gcs entities? if not, then we shouldn't be using a schema_resolver

Reply:
yes we do have schema for gcs, but this point is also valid i ll use gcs schema resolver when its required i ll remove this code from here